src/Controller/AutoSegmentationController.py

```diff
@@ -40,20 +54,67 @@ def start_button_clicked(self) -> None:
 
         self.run_task("total", self._view.get_segmentation_roi_subset())
 
+    def save_button_clicked(self, save_name: str, save_list: list[str]) -> None:
+        """
+        To be called when the button to save the selected segmentation task is clicked
+        :rtype: None
+        """
+        self.database.insert_row(save_name, save_list)
+
+    def load_button_clicked(self, value: str) -> None:
+        """
+        To be called when the button to load the selected saved segmentation is clicked
+        :rtype: None
+        """
+        self.database.select_entry(value)
+        self._view.load_selection(self.database.select_entry(value))
+
+    def delete_button_clicked(self, value: str) -> None:
+        """
+        To be called when the button to delete the selected segmentation task is clicked
+        :rtype: None
+        """
+        if value in self.save_list:
+            self.database.delete_entry(value)
+            self._view.remove_save_item(value)
+
+    def database_save_list(self) -> None:
+        """
+        Gets a list of all saved segmentaion choices and ads them to the Selector widget
+        :rtype: None
+        """
+        self.save_list: list[str] = self.database.get_save_list()
+        self._view.add_save_list(self.save_list)
+
     def show_view(self):
         """
         To Display the view on Screen
         :rtype: None
         """
         if self._view is None:
             self._view = AutoSegmentWindow(self.view_state)
+        self.database_save_list()
         self._view.show()
 
         # Bring window to front and make active (if already visible)
         if self._view.isVisible():
             self._view.raise_()
             self._view.activateWindow()
 
+    def update_database_feedback(self, text: str) -> None:
```

Comment:
is this feedback status information?  error responses?  And transmit where?
It looks like it is intended to provide the user with visual/text information (in the view) regarding warning or error level responses when an update to the database took place.  Is this "most recent", an accumulation or something else?